assets/js/actions/index.js

```diff
@@ -14,4 +14,20 @@ export const GameEvents = Object.freeze({
   ROUND_STATE_FETCHED: 'ROUND_STATE_FETCHED',
   PLAYER_MOVED: 'PLAYER_MOVED',
   SHOW_RESULT: 'SHOW_RESULT'
-})
\ No newline at end of file
+})
+
+
+export const setGameId = function (gameRoundFromServer, currentGameRound){
+  return {
+      game_id: gameRoundFromServer.game_id,
+      board: currentGameRound.board
+  }
+}
+
+export const updateGameRound = function (gameRoundFromServer){
```

Comment:
This one does not update anything but just create a copy. 
You could do this:

const copyGameRound = (round) => {...round}